Lombiq.Tests.UI/Models/VisualVerificationMatchConfiguration.cs

```diff
@@ -13,6 +16,11 @@ public class VisualVerificationMatchConfiguration<TSelf>
     /// </summary>
     public string FileNameSuffix { get; private set; }
 
+    /// <summary>
+    /// Gets the list of <see cref="PlatformID"/> where the test can be run. If null, all platforms are selected.
```

Comment:
Don't use `null` as an empty enumerable. Accept null but provide an empty one by default, and both should mean empty.